cdap-cli/src/main/java/co/cask/cdap/cli/command/SetProgramRuntimeArgsCommand.java

```diff
@@ -56,14 +54,14 @@ public void perform(Arguments arguments, PrintStream output) throws Exception {
     String runtimeArgsString = arguments.get(ArgumentName.RUNTIME_ARGS.toString());
     Map<String, String> runtimeArgs = ArgumentParser.parseMap(runtimeArgsString);
     programClient.setRuntimeArgs(programId, runtimeArgs);
-    output.printf("Successfully set runtime args of %s '%s.%s' of application '%s' to '%s'\n",
+    output.printf("Successfully set runtime args of %s '%s' of application '%s.%s' to '%s'\n",
                   elementType.getName(), programName, appName, appVersion, runtimeArgsString);
   }
 
   @Override
   public String getPattern() {
-    return String.format("set %s runtimeargs <%s> <%s>", elementType.getShortName(),
-                         elementType.getArgumentName(), ArgumentName.RUNTIME_ARGS);
+    return String.format("set %s runtimeargs <%s> [version <%s>] <%s>", elementType.getShortName(),
```

Comment:
Yes. All the tests like `testCommandOutputContains(cli, "start service " + serviceName, "Successfully started service")` with `serviceName` doesn't contain the optional version argument